test/modules/minifySvg.js

```diff
@@ -54,4 +69,21 @@ describe('minifySvg', () => {
             {minifySvg: maxPreset.minifySvg}
         );
     });
+
+    // https://github.com/posthtml/htmlnano/issues/88
+    it('should work with <svg> with <script> inside (issue #88)', () => {
+        return init(
+            svgContainsCDATA,
+
+            '<svg xmlns="http://www.w3.org/2000/svg"><style>.label{color:red}</style><text class="label">example</text><text class="label">example</text></svg>\n    <svg xmlns="http://www.w3.org/2000/svg"><script>/*<![CDATA[*/const x="<>";/*]]>*/</script></svg>',
```

Comment:
The `CDATA` inside `<style>` here has been stripped by SVGO, because SVGO determines that there is no CSS that needs to be escaped, thus no `CDATA` is required.